app/Http/Requests/Request.php

```diff
@@ -0,0 +1,9 @@
+<?php namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+abstract class Request extends FormRequest {
+
+	//
```

Comment:
This abstraction does not do anything, but specify a new type. Please add a class comment explaning its purpose.